.drone.yml

```diff
@@ -374,7 +374,17 @@ pipeline:
     when:
       branch: [dev]
       event: [push]
-
+  
+  build_solidity_docs:
```

Comment:
Why do we build the docs in drone? They're not being pushed anywhere. Maybe we can do this step before the `assert_no_changes`, so we make sure the docs are up to date.

Reply:
I can move `build_Solidity_docs` before `go_generate` as there is `make assert_no_changes`.